docs/content/3.manual/manual.yml

```diff
@@ -408,15 +412,16 @@ sections:
           Like `.[]`, but no errors will be output if . is not an array
           or object.
 
-      - title: "`,`"
+      - title: "Comma, Output Stream Join: `,`"
```

Comment:
A one word, precise name is not really necessary. Your point about opcodes and names is valid, but would like to point out that "each" is just another way of expressing iteration. I actually think "Fork" wouldn't be a bad candidate for a one-word, precise name. Your suggestion of "Expression Concat" is another good choice, but less desirable, imo.